index.html

```diff
@@ -2573,18 +2603,23 @@ <h5>Uses beyond mapping</h5>
 
 <h5>Related web specifications</h5>
 <p>
-<!-- Any existing/proposed specs that address similar needs or otherwise interact.
-      Include links with proper ReSpec references. -->
+The <a data-cite="SVG2#">SVG</a> <code>image</code> element provides an example
+of external image content being rendered into a specified rectangle within a coordinate system.
+</p>
+<p>
+[[[Canvas]]] includes a JavaScript method, <code>drawImage()</code>
+for embedding external images within the canvas element.
```

Comment:
```suggestion
Similarly, the <a data-cite="HTML#the-canvas-element">HTML Canvas2D API</a> has a <code>drawImage()</code> method
for rendering a scaled and positioned copy of an external image within the canvas. [[SVG2]][[HTML]]
```

(You'll want to test that that actually does the right thing in ReSpec. Not sure I've tested the auto-linking to multipage specs.)

Reply:
Fun fact, both `<a data-cite="HTML#the-canvas-element">` and `<a data-cite="HTML#canvas">` work as intended!